--- ./configure.in.org	Thu Aug  8 19:37:38 2002
+++ ./configure.in	Thu Aug  8 19:38:02 2002
@@ -1,78 +0,0 @@
-dnl Process this file with autoconf to produce a configure script.
-AC_INIT(3ddesk.cpp)
-AM_INIT_AUTOMAKE(3ddesktop,0.1.5)
-
-dnl Checks for programs.
-AC_PROG_CXX
-AC_PROG_CC
-
-dnl Checks for header files.
-AC_PATH_XTRA
-
-AC_HEADER_DIRENT
-AC_HEADER_STDC
-AC_CHECK_HEADERS(fcntl.h sys/time.h unistd.h)
-dnl AC_CHECK_HEADERS(Imlib2.h)
-
-dnl Checks for libraries.
-MDL_HAVE_OPENGL
-
-AC_CHECK_HEADERS(GL/gl.h, ,exit)
-AC_CHECK_HEADERS(X11/extensions/xf86vmode.h, ,exit)
-
-dnl ---- I'll assume the header check is enough
-dnl AC_CHECK_LIB(Xxf86vm, XF86VidModeQueryVersion)
-dnl AC_CHECK_LIB(Xxf86vm, XF86VidModeQueryVersion, ,exit)
-
-dnl AC_CHECK_LIB(Imlib2, imlib_create_image_from_drawable)
-
-IMLIB2_CONFIG_IN_PATH=yes
-AC_PROGRAM_CHECK(IMLIB2_CONFIG_IN_PATH, imlib2-config, yes, no)
-if test $IMLIB2_CONFIG_IN_PATH = no; then
-    echo "ERROR."
-    echo "The imlib2-config development script was not found in your execute"
-    echo "path."
-    echo "You may have imlib2 installed somewhere not covered by your path."
-    echo ""
-    echo "If this is the case make sure you have the packages installed, AND"
-    echo "that the imlib2-config script is in your execute path (see your"
-    echo "shell's manual page on setting the \$PATH environment variable)."
-    AC_MSG_ERROR([Fatal Error: no imlib2-config detected.])
-    exit;
-fi
-imlib2_cflags=`imlib2-config --cflags`
-imlib2_libs=`imlib2-config --libs`
-AC_SUBST(imlib2_cflags)
-AC_SUBST(imlib2_libs)
-
-
-
-dnl AC_CHECK_LIB(GL, glClearColor)
-
-dnl AC_CHECK_LIB(GLU, gluBuild2DMipmaps)
-
-dnl AC_CHECK_LIB(X11, XAllowEvents)
-
-dnl AC_CHECK_LIB(Xi, XQueryTree)
-
-dnl AC_CHECK_LIB(Xmu, XInternAtom)
-
-dnl AC_CHECK_LIB(glut, glutSwapBuffers)
-
-AC_CHECK_LIB(m, cos)
-
-dnl Checks for typedefs, structures, and compiler characteristics.
-AC_C_INLINE
-AC_HEADER_TIME
-
-dnl Checks for library functions.
-AC_CHECK_FUNCS(gettimeofday mkdir strerror)
-
-AC_OUTPUT(Makefile)
-
-dnl AC_MSG_RESULT([
-dnl Using the following for compiling and linking OpenGL programs:
-dnl 
-dnl   GL_CFLAGS: $GL_CFLAGS
-dnl     GL_LIBS: $GL_LIBS
-dnl ])
--- ./configure.ac.org	Thu Aug  8 19:38:14 2002
+++ ./configure.ac	Thu Aug  8 21:06:00 2002
@@ -0,0 +1,80 @@
+dnl Process this file with autoconf to produce a configure script.
+AC_INIT([3ddesktop],[0.1.5],[your@mail.org])
+AC_CONFIG_SRCDIR([3ddesk.cpp])
+AM_INIT_AUTOMAKE
+
+dnl Checks for programs.
+AC_PROG_CC
+AC_PROG_CXX
+AC_C_INLINE
+AC_HEADER_TIME
+
+dnl Checks for header files.
+AC_PATH_XTRA
+
+CPPFLAGS="-I/usr/X11R6/include"
+AC_HEADER_DIRENT
+AC_HEADER_STDC
+AC_CHECK_HEADERS([fcntl.h sys/time.h unistd.h])
+dnl AC_CHECK_HEADERS(Imlib2.h)
+
+dnl Checks for libraries.
+MDL_HAVE_OPENGL
+
+AC_CHECK_HEADERS(GL/gl.h,[ ] ,[AC_MSG_ERROR([You need gl.h in your system])])
+AC_CHECK_HEADERS(X11/extensions/xf86vmode.h,[ ] ,[AC_MSG_ERROR([You need xf86vmode.h in your system])])
+
+dnl ---- I'll assume the header check is enough
+dnl AC_CHECK_LIB(Xxf86vm, XF86VidModeQueryVersion)
+dnl AC_CHECK_LIB(Xxf86vm, XF86VidModeQueryVersion, ,exit)
+
+dnl AC_CHECK_LIB(Imlib2, imlib_create_image_from_drawable)
+IMLIB2_CONFIG_IN_PATH=yes
+AC_CHECK_PROG(IMLIB2_CONFIG_IN_PATH, imlib2-config, yes, no)
+if test $IMLIB2_CONFIG_IN_PATH = no; then
+    echo "ERROR."
+    echo "The imlib2-config development script was not found in your execute"
+    echo "path."
+    echo "You may have imlib2 installed somewhere not covered by your path."
+    echo ""
+    echo "If this is the case make sure you have the packages installed, AND"
+    echo "that the imlib2-config script is in your execute path (see your"
+    echo "shell's manual page on setting the \$PATH environment variable)."
+    AC_MSG_ERROR([Fatal Error: no imlib2-config detected.])
+    exit;
+fi
+imlib2_cflags=`imlib2-config --cflags`
+imlib2_libs=`imlib2-config --libs`
+AC_SUBST(imlib2_cflags)
+AC_SUBST(imlib2_libs)
+
+#AC_CHECK_PROG(IMLIB2_CONFIG_IN_PATH, imlib2-config,
+#		 CFLAGS="$CFLAGS `imlib2-config --cflags`"
+#		 LIBS="$LIBS `imlib2-config --libs`",
+#		 [AC_MSG_ERROR(Fatal Error: no imlib2-config detected.)])
+
+dnl AC_CHECK_LIB(GL, glClearColor)
+
+dnl AC_CHECK_LIB(GLU, gluBuild2DMipmaps)
+
+dnl AC_CHECK_LIB(X11, XAllowEvents)
+
+dnl AC_CHECK_LIB(Xi, XQueryTree)
+
+dnl AC_CHECK_LIB(Xmu, XInternAtom)
+
+dnl AC_CHECK_LIB(glut, glutSwapBuffers)
+
+AC_CHECK_LIB(m, cos)
+
+dnl Checks for library functions.
+AC_CHECK_FUNCS([gettimeofday mkdir strerror])
+
+AC_OUTPUT([Makefile])
+
+dnl AC_MSG_RESULT([
+dnl Using the following for compiling and linking OpenGL programs:
+dnl 
+dnl   GL_CFLAGS: $GL_CFLAGS
+dnl     GL_LIBS: $GL_LIBS
+dnl ])
--- ./Makefile.in.org	Thu Aug  8 21:03:17 2002
+++ ./Makefile.in	Thu Aug  8 21:07:15 2002
@@ -98,13 +98,13 @@
 
 maintainer-clean: clean
 
-# automatic re-running of configure if the configure.in file has changed
-${srcdir}/configure: configure.in aclocal.m4
+# automatic re-running of configure if the configure.ac file has changed
+${srcdir}/configure: configure.ac aclocal.m4
 	cd ${srcdir} && autoconf
 
 # autoheader might not change config.h.in, so touch a stamp file
 ${srcdir}/config.h.in: stamp-h.in
-${srcdir}/stamp-h.in: configure.in #aclocal.m4
+${srcdir}/stamp-h.in: configure.ac #aclocal.m4
 		cd ${srcdir} && autoheader
 		echo timestamp > ${srcdir}/stamp-h.in
 
