--- 3ddesktop-0.2.0/3ddeskd.cpp.wiget	Thu Aug  8 03:21:43 2002
+++ 3ddesktop-0.2.0/3ddeskd.cpp	Mon Aug 12 12:26:11 2002
@@ -1773,7 +1773,7 @@
 
     if (!load_image(filename, data, &sizex, &sizey, 1)) {
 
-        sprintf (filename, "/usr/local/share/3ddesktop/digits.bmp");
+        sprintf (filename, SHAREDIR "/digits.bmp");
 
         if (!load_image(filename, data, &sizex, &sizey, 1)) {
 
--- 3ddesktop-0.2.0/config.hpp.wiget	Thu Aug  8 03:21:13 2002
+++ 3ddesktop-0.2.0/config.hpp	Mon Aug 12 12:26:11 2002
@@ -54,7 +54,7 @@
            "                        priceisright, flip, or random)\n"
            "    --acquire[=#]    Grab images for all the desktops by cycling thru\n"
            "                       (sleep for # millisecs at each screen for refresh)\n"
-           "    --dir=/path      the install dir if different than /usr/share/3ddesktop\n"
+           "    --dir=/path      the install dir if different than " SHAREDIR "\n"
            "    --workspaces     switch between workspace desktops instead of grid areas\n"
            "    --depth=#        specify how much to zoom out\n"
            "    --randdelay=#    seconds before random movement (0 for none) 15 default\n"
@@ -144,7 +144,7 @@
         goto_column = NO_GOTO;
         goto_row = NO_GOTO;
 
-        strcpy(base_dir, "/usr/share/3ddesktop");
+        strcpy(base_dir, SHAREDIR);
 
         digit_size = 50;
         digit_color = COLOR_RED;
@@ -508,21 +508,15 @@
         FILE *fp = fopen(config_file, "r");
         if (!fp) {
 
-            sprintf(config_file, "/usr/local/share/3ddesktop/3ddesktop.conf");
+            sprintf(config_file, SYSCONFDIR "/3ddesktop.conf");
             
             fp = fopen(config_file, "r");
             if (!fp) {
                 
-                sprintf(config_file, "/usr/share/3ddesktop/3ddesktop.conf");
-        
-                fp = fopen(config_file, "r");
-                if (!fp) {
-                
-                    // don't error on this - may not be there
-                    msgout (DEBUG, "Could not open config file: 3ddesktop.conf: %s\n", 
-                            strerror(errno));
-                    return;
-                }
+                // don't error on this - may not be there
+                msgout (DEBUG, "Could not open config file: 3ddesktop.conf: %s\n", 
+                        strerror(errno));
+                return;
             }
         }
 
--- 3ddesktop-0.2.0/Makefile.in.wiget	Mon Aug 12 12:26:11 2002
+++ 3ddesktop-0.2.0/Makefile.in	Mon Aug 12 12:26:41 2002
@@ -33,17 +33,20 @@
 CC = @CC@
 CXX = @CXX@
 
-CFLAGS = -Wall -I. @CFLAGS@ @imlib2_cflags@
-#-I/usr/include/kde -I/usr/lib/qt-2.3.0/include 
-CPPFLAGS = -g -Wall -O2 @X_CFLAGS@ @GL_CFLAGS@ @CPPFLAGS@ @imlib2_cflags@
 LDFLAGS = @LDFLAGS@ 
 LIBS = @LIBS@ @GL_LIBS@ @X_LIBS@ -lXxf86vm @imlib2_libs@
 INSTALL = install
 prefix = @prefix@
-bindir = $(prefix)/bin
-libdir = $(prefix)/lib
-infodir = $(prefix)/info
-sharedir = $(prefix)/share/3ddesktop
+bindir = @bindir@
+libdir = @libdir@
+infodir = @infodir@
+datadir = @datadir@
+sharedir = $(datadir)/3ddesktop
+sysconfdir = @sysconfdir@
+
+CFLAGS = $(OPT) -Wall -I. @CFLAGS@ @imlib2_cflags@ 
+#-I/usr/include/kde -I/usr/lib/qt-2.3.0/include 
+CPPFLAGS = -Wall @X_CFLAGS@ @GL_CFLAGS@ @CPPFLAGS@ @imlib2_cflags@ -DSHAREDIR=\"$(sharedir)\" -DSYSCONFDIR=\"$(sysconfdir)\"
 
 SOURCES=3ddeskd.cpp xutil.cpp faceset.cpp util.cpp 
 DOCS=README ChangeLog
@@ -54,12 +57,13 @@
 all: 3ddeskd 3ddesk
 
 install: all
-	$(INSTALL) -d $(DESTDIR)$(bindir) $(DESTDIR)$(sharedir)
+	$(INSTALL) -d $(DESTDIR)$(bindir) $(DESTDIR)$(sharedir) $(DESTDIR)$(sysconfdir)
 	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(bindir)
 	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(sharedir)
 	$(INSTALL) 3ddeskd $(DESTDIR)$(bindir)/3ddeskd
 	$(INSTALL) 3ddesk $(DESTDIR)$(bindir)/3ddesk
-	$(INSTALL) digits.bmp 3ddesktop.conf $(DESTDIR)$(sharedir)
+	$(INSTALL) digits.bmp $(DESTDIR)$(sharedir)
+	$(INSTALL) 3ddesktop.conf $(DESTDIR)$(sysconfdir)
 
 uninstall:
 	/bin/rm -f $(bindir)/3ddeskd $(bindir)/3ddesk
